packages/webapp/src/components-v2/ui/tooltip.tsx

```diff
@@ -19,20 +22,50 @@ function TooltipTrigger({ ...props }: React.ComponentProps<typeof TooltipPrimiti
     return <TooltipPrimitive.Trigger data-slot="tooltip-trigger" {...props} />;
 }
 
-function TooltipContent({ className, sideOffset = 0, children, ...props }: React.ComponentProps<typeof TooltipPrimitive.Content>) {
+const tooltipContentVariants = cva(
+    'text-text-secondary animate-in fade-in-0 zoom-in-95 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 z-50 w-fit origin-(--radix-tooltip-content-transform-origin) rounded-md px-3 py-1.5 text-s leading-4 text-balance',
+    {
+        variants: {
+            variant: {
+                primary: 'bg-bg-subtle',
+                secondary: 'bg-bg-surface'
```

Comment:
This is basically a black tooltip (so that it can be used on top of a light grey surface). Open to a better name than `secondary` :)